### DISCUSSION TITLE
Switch Charge/Discharge on and off using BMS software switches

### DISCUSSION THREAD

**Discussion by squowse** [Louisvdw/dbus-serialbattery#1028] [Ideas]
Do you think this would ever be possible using Victron Venus OS? 2 way communication?
I use JBD BMS mostly.

**Comment by AlexdeKairos** [comment]
Actually with my JBD BMS, I'm already able to switch on and off my charge and discharge switches from cerbo gx. Also the balancing of the cels. So, just review your menus.  Under IO menu you should have the switches.

**Reply by squowse** [reply]
Thanks I’ll have to look more closely
